src/View/Parsers/Diff.php

```diff
@@ -65,94 +64,29 @@ public static function compareHTML($from, $to, $escape = false)
         $set2 = self::getHTMLChunks($to);
 
         // Diff that
-        $diff = new Diff($set1, $set2);
-
-        $tagStack[1] = $tagStack[2] = 0;
-        $rechunked[1] = $rechunked[2] = [];
-
-        // Go through everything, converting edited tags (and their content) into single chunks.  Otherwise
-        // the generated HTML gets crusty
```

Comment:
Note that this _is_ probably important to keep in some form - I think it's mean to guard against incorrectly breaking up html tags like so:

```diff
- <div>
+ <span>
  Some content here
- </div>
+ </span>
```
which looks right when viewed in a diff syntax highlighter, but the html for it would end up looking like this:
```html
<del><div></del><ins><span></ins>
  Some content here
<del></div></del><ins></span></ins>
```

I think this rechunking would ultimately result in this, would is correct:
```html
<del><div>
  Some content here
</div></del>
```html
<ins><span>
  Some content here
</span></ins>
```

Though that's just guessing - I haven't validated this assumption.
```

Reply:
Yup we needed to keep it, albeit in a refactored state.